### ISSUE TITLE
Doesn't work with big integers

### ISSUE THREAD

**Issue by dpsanders** [dpsanders/ValidatedNumerics.jl#58]
```
julia> ValidatedNumerics.make_interval(big(1_000_000_000_000_000_000_000_000_000_000))
ERROR: MethodError: `make_interval` has no method matching make_interval(::BigInt)
```

**Comment by dpsanders** [comment]
The reason is:
```
julia> parse("12312335341234123423413412341234123423")
:(@int128_str "12312335341234123423413412341234123423")

julia> parse("12312335341234123423413412341234123423123410293847123904817234")
:(@big_str "12312335341234123423413412341234123423123410293847123904817234")
```

This will be have to be taken account by hand in `transform`.

**Comment by lbenet** [comment]
Three short comments: 

- `make_interval` *does* work; you simply forgot to specify the type of interval:
```julia
julia> set_bigfloat_precision(53)
53
julia> ValidatedNumerics.make_interval(BigFloat,big(1_000_000_000_000_000_000_000_000_000_000))
[9.9999999999999988e+29, 1.0000000000000000e+30]₅₃
```

- It's true that `@interval(12312335341234123423413412341234123423)` does not work...

- What it does work is:
```julia
julia> x = 1_000_000_000_000_000_000_000_000_000_000
1000000000000000000000000000000

julia> @interval(x)
[1.0e30, 1.0e30]
```

**Comment by lbenet** [comment]
I think this should go to the docs, emphasizing [the way](https://github.com/dpsanders/ValidatedNumerics.jl/issues/58#issuecomment-142459993) in which this can be implemented.

**Comment by dpsanders** [comment]
I think we just should implement it. I'll do it at some point.

**Comment by lbenet** [comment]
I think it all boils down to implementing `big(a::Interval)` and other functions of the same kind. The rest is the done by `transform`.

**Comment by dpsanders** [comment]
It's a bit more subtle than that, since as  I showed in https://github.com/dpsanders/ValidatedNumerics.jl/issues/58#issuecomment-142452078
big integers are parsed in a certain way. This will have to be taken into account explictly in `transform`.

**Comment by dpsanders** [comment]
Implemented in #98.
